pkg/publish/daemon.go

```diff
@@ -1,16 +1,18 @@
-// Copyright 2018 Google LLC All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/*
+Copyright 2021 Google LLC All Rights Reserved.
```

Comment:
We should fix this. We don't ever want to bump this year.

Reply:
> Should we consider updating hack/boilerplate/boilerplate.go.txt to match this?

I think this becomes easier to do once https://github.com/google/ko/pull/364 gets merged, I just haven't done anything out of laziness. I'm fine with whatever is most idiomatic or easiest :P